packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -328,20 +326,19 @@ const makeRenderer = (renderConfig: {
 		return routesPromise.then(resolvedRoutes =>
 			initializeStore(resolvedRoutes).then(store => {
 				// the initial addFlags call will only be key'd by member ID
-				return addFlags(store, { id: parseMemberCookie(state).id })
+				return addFlags(store, { id: parseMemberCookie(request.state).id })
```

Comment:
was this a bug before? or am i missing where this change is coming from?

Reply:
`state` was assigned as a `const` destructured from `request` in the old code (on line 228), but I got rid of that destructuring because it added a lot more variable names to keep track of in this scope, and `state` is ambiguous in code that touches both the Hapi `request` _and_ the Redux store - I wanted to make the reference more explicitly tied to the `request` parent.